lib/travis/build/addons/rethinkdb.rb

```diff
@@ -0,0 +1,44 @@
+require 'shellwords'
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Rethinkdb < Base
+        SUPER_USER_SAFE = true
+
+        RETHINKDB_GPG_KEY = '0x3A8F2399'
+
+        def after_prepare
+          sh.fold 'rethinkdb' do
+            sh.if "ENV['TRAVIS_OS_NAME'] != 'linux'" do
+              sh.echo "The RethinkDB addon only works on Linux. $TRAVIS_OS_NAME is #{ENV['TRAVIS_OS_NAME']}.", ansi: :red
```

Comment:
RethinkDB appears to work on the Mac, so I am not sure why you want to limit the scope.